src/site/content/en/blog/rakuten/index.md

```diff
@@ -0,0 +1,166 @@
+
+Understanding the impact of web performance on user experience, the Rakuten 24 team has been continuously measuring, optimizing and monitoring Core Web Vitals and other metrics.
+
+As a result, **over 75% of their users are experiencing good Largest Contentful Paint (LCP), First Input Delay (FID) and First Contentful Paint (FCP)**.
+
+After analyzing homepage data, Rakuten 24 found that a good LCP score can lead to an increase of up to **61.13% in conversion rate**, **26.09% in revenue per visitor**, and **11.26% in average order value**. A good FID score can lead to an increase of up to **55.88% in conversion rate**.
+
+To further correlate Core Web Vitals and business metrics, Rakuten 24 also ran an A/B test focused on optimizing Core Web Vitals and related metrics, and saw an improvement of **53.37% in revenue per visitor**, **33.13% in conversion rate**, **15.20% in average order value**, **9.99% in average time spent**, and a **35.12% reduction in exit rate**.
+
+## Highlight the opportunity
+
+Though optimizing web performance is a smart investment to improve user experience and business growth, Rakuten 24 team understands how difficult it can be to convince stakeholders to adopt Core Web Vitals and focus on web performance. They believe that showing stakeholders exactly what kind of Return on Investment (ROI) performance optimization can bring is the best way to get them on board.
+
+Being a relatively new and independent service, Rakuten 24 used their advantage of flexibility to take up the challenge. They believe that the result of their case study would help them make more data-oriented decisions in the future, as well as help other developers measure the impact of their work and convince their stakeholders that improving performance is worth the investment.
+
+<figure>
+  {% Img src="image/jL3OLOhcWUQDnR4XjewLBx4e3PC3/ij6fET9W23vj1IOb3e6O.jpg", alt="Example screenshots of Rakuten 24's homepage, with mobile device frames around each screenshot.", width="800", height="537" %}
+  <figcaption>
+    Some example screenshots of Rakuten 24's homepage.
+  </figcaption>
+</figure>
+
+## The approach they used
+
+### Web performance optimization
+
+#### Optimize JavaScript and resources
+
+- Eliminate render-blocking resources.
+- Split code and use [dynamic `import()`](https://v8.dev/features/dynamic-import).
+- Split all content into separate parts and lazy load below-the-fold HTML files.
+- Execute and load JavaScript on demand.
+- Identify slow JavaScript resources and optimize the loading process by using the async attribute on `<script>` tags and establishing early connections to important origins (Resource hints such as `dns-prefetch`, `preconnect`, and `preload`).
+- [Remove unused code](/remove-unused-code/), and [minify and compress code](/reduce-network-payloads-using-text-compression/).
+- Use a CDN.
```

Comment:
@malchata I'm wondering if this should be `[Use a CDN](/content-delivery-networks/)` or did you leave the URL out on purpose? If you left it out on purpose, that's fine too. Just checking in case!

Reply:
Good catch. I updated this in a commit.